_maps/shrike.json

```diff
@@ -11,5 +11,18 @@
 	    "emergency": "emergency_void"},
     "mine_disable": 1,
 	"ship_type": "/obj/structure/overmap/nanotrasen/missile_cruiser/starter/shrike",
-	"starmap_path": "config/starmap/starmap.json"
+	"starmap_path": "config/starmap/starmap.json",
+
+	"map_description": "The Shrike was once in service some time ago but has recently been unearthed as a schematic for cheap, fast ships perfect for use as a QRF. Most if not all Shrike-Class ships have been abandoned for some time in one shipyard or another until Skink Industries bought them en masse and outfitted them to the bare minimum requirements. As a result, Shrike vessels lack some amenities and technologies though with plenty of room for crew improvements.",
+	"manufacturer": "Skink Industries",
```

Comment:
Are we really going with the shameless self-insert because I couldn't remember any of the in-lore companies?

Reply:
I mostly just did whatever was in the document, feel free to suggest stuff or just tell me to look at the wiki lol

Reply:
I don't care if you just check it out and push changes either but also I'll sort comments out when I get home from pinball